data/config/serverdb.kvc

```diff
@@ -407,9 +407,9 @@ NServers=1
 0_Hostname=irc.forestnet.org
 0_Description=Forestnet:%20Random%20server
 NServers=1
-[Freenode]
+[freenode]
```

Comment:
im not sure if any stuff like favorites or network-based autoconnect is dependent on the capitalisation of the name...